V1/Constants/Messages.cs

```diff
@@ -39,7 +38,7 @@ public class Messages
         public static readonly string Token_OTPExpired = $"OTP has expired.";
         public static readonly string Token_OTPFailed = $"OTP is invalid.";
         public static readonly string Token_OTPThreshold = $"Too many OTPs."; //TODO: Copy change
-        public static readonly string Token_InvaldPayload = $"OTP Payload is invalid."; //TODO: Copy change
+        public static readonly string Token_InvaldPayload = $"OTP Payload is invalid."; 
```

Comment:
all/most these strings doesn't need interpolation, why is initialised as such?

Reply:
I think it was just a bad copy and past, i will remove it